Project1/AdvancedCalculator.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Project1
+{
+    internal class AdvancedCalculator
+    {
+        public bool isPrimeNumber(int value1)
```

Comment:
Tutaj nie jestem pewny czy ten algorytm działa, a na pewno nei optymalnie. To idzie zrobić robiąc jednego fora  i wnim ifa sprawdzającego czy liczba mniejsza od podanej będzie podzielna. Potem zmieniasz zmienną bool na false np kiedy nie jest podzielna czy coś w tym stylu.